### ISSUE TITLE
Pururin: Domain changed and pages are encoded

### ISSUE THREAD

**Issue by AimannQn** [tachiyomiorg/tachiyomi-extensions#8198]
### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I have updated the app to version **[0.11.1](https://github.com/tachiyomiorg/tachiyomi/releases/latest)**.
- [x] I have updated all installed extensions.
- [x] I have tried the [troubleshooting guide](https://tachiyomi.org/help/guides/troubleshooting/).
- [X] If this is an issue with the app itself, I should be opening an issue in the [app repository](https://github.com/tachiyomiorg/tachiyomi/issues/new/choose).
- [X] I will fill out all of the requested information in this form.

### Tachiyomi version

Stable 0.11.1

### Android version

Android 11

### Source information

Pururin 1.2.3

### Steps to reproduce

1. Open Pururin
2. Read Doujin

### Expected behavior

Can Read Doujin

### Actual behavior

When I want to read doujin It say 'No Pages Found', It happen after the Website got fix from being Down, It need an Update

### Other details

Basically Entire Doujin from Pururin Cant be read, Everytime you click read it say That
![Screenshot_20210718-053609_Tachiyomi](https://user-images.githubusercontent.com/84202326/126297307-b96a9fae-b529-4228-a4c1-e1f57ca83d96.jpg)

**Comment by Safuez** [comment]
Found this on Tachiyomi sy if it's help![_20210720_223336.JPG](https://user-images.githubusercontent.com/81817527/126353119-6c2ccc19-afce-44b8-9232-68b85e786e55.JPG)

**Comment by AimannQn** [comment]
> Found this on Tachiyomi sy if it's help![_20210720_223336.JPG](https://user-images.githubusercontent.com/81817527/126353119-6c2ccc19-afce-44b8-9232-68b85e786e55.JPG)
Also got the same thing on TachiJ2K, I guess it need an Extension Update after the Website got fix from being down

**Comment by Clouddark75** [comment]
They changed of domain and are using a basic Base64 encryption for the chapter pages now.

**Comment by AimannQn** [comment]
> They changed of domain and are using a basic Base64 encryption for the chapter pages now.

Sorry but what does it mean, and can I still read the Pururin Doujin?

**Comment by jobobby04** [comment]
It means you need to wait for an update, we need to account for Pururin's new changes

**Comment by AimannQn** [comment]
> It means you need to wait for an update, we need to account for Pururin's new changes

Oooh.. Thanks for the info!! also who is the one that make extension Updated? is it like the website extension 
 Mod/Owner?

**Comment by AimannQn** [comment]
> Anyone with time and knowledge can update the Tachi extension, this is a community project

Ooh... Thanks For the Info Brother!!

**Comment by alessandrojean** [comment]
/edit-title "Puririn: Domain changed and pages are encoded"

**Comment by alessandrojean** [comment]
/edit-title "Pururin: Domain changed and pages are encoded"
